config/routes.rb

```diff
@@ -16,7 +16,9 @@
   put 'user_reports/undo_delete/:id' => 'user_reports#undo_delete', as: :undo_delete_users_report
 
   resources :pages, only: [:index, :new, :create, :edit]
-  resources :volunteer_ops, :only => [:index, :edit, :show, :update, :destroy]
+  resources :volunteer_ops, :only => [:index, :edit, :show, :update, :destroy] do
+    get 'search', on: :collection
```

Comment:
if that's just reorganizing how they appear in rake routes then that sounds reasonable - want to open a separate ticket for refactoring that?   If so, please be sure to include copies of before and after on the rake routes (and note any other side effects)